app/build.gradle

```diff
@@ -372,14 +372,26 @@ dependencies {
     testImplementation 'org.robolectric:shadows-multidex:4.4'
 
     implementation 'com.github.bumptech.glide:compose:1.0.0-alpha.5'
-    implementation 'androidx.compose.ui:ui:1.5.2'
-    implementation 'androidx.compose.ui:ui-tooling:1.5.2'
+    implementation 'androidx.compose.ui:ui:1.6.2'
+    implementation 'androidx.compose.animation:animation:1.6.2'
+    implementation 'androidx.compose.ui:ui-tooling:1.6.2'
+    implementation "androidx.compose.runtime:runtime-livedata:1.6.2"
+    implementation 'androidx.compose.foundation:foundation-layout:1.6.2'
+    implementation 'androidx.compose.material:material:1.6.2'
+    androidTestImplementation 'androidx.compose.ui:ui-test-junit4-android:1.6.2'
+    debugImplementation 'androidx.compose.ui:ui-test-manifest:1.6.2'
+
     implementation "com.google.accompanist:accompanist-themeadapter-appcompat:0.33.1-alpha"
+    implementation "com.google.accompanist:accompanist-pager:0.33.1-alpha"
```

Comment:
Accompanist Pager and PagerIndicator are deprecated.
We need to use the classes from `androidx.compose.foundation.pager`

Reply:
gone